src/main/java/com/piekoszek/nowaksharedrent/jwt/JwtData.java

```diff
@@ -11,6 +12,7 @@
 @Getter
 @Builder
 @EqualsAndHashCode
+@AllArgsConstructor
```

Comment:
@Builder is not enough?

Reply:
I will check it today but as far as I remember there was error during building an app saying that default constructor is missing.

Reply:
Yup, builder was enough.